Makefile

```diff
@@ -0,0 +1,33 @@
+SHELL := /bin/bash
+
+DST := $(patsubst %.bs,%.html,$(wildcard *.bs))
+REMOTE := $(filter remote,$(MAKECMDGOALS))
+
+all: $(DST)
+	@ echo "All done"
+
+%.html : %.bs node_modules/vnu-jar/build/dist/vnu.jar
+ifndef REMOTE
+	@ echo "Building $@"
+	bikeshed --die-on=warning spec $< $@
+	java -jar node_modules/vnu-jar/build/dist/vnu.jar --also-check-css $@
```

Comment:
Is this new dependency on `vnu.jar` necessary?

Reply:
Our CI checks it, so I figured we could make it easy to run locally.

Here's an example of a failure in a previous PR of mine: https://github.com/WebAudio/web-speech-api/actions/runs/13461066795?pr=144 (which was legitimate, my markup was incorrect).